### DISCUSSION TITLE
Adding notes to releases after publishing

### DISCUSSION THREAD

**Discussion by pcuellar** [knowledgefutures/pubpub#1287] [Support] [answered]
Hello, 

I misunderstood the release notes and thought that notes submitted on "request publication" will be added as part of the release notes. As a result, I ended up releasing without notes. Is there a way to add these notes now that the release has gone live? 

Thanks

**Comment by gabestein** [comment, marked as answer]
Hello! Unfortunately, release notes are not editable. But you could make a small change (even add a space, etc.) and create a second release with the notes if you'd like.
